src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,36 @@
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  filterStatus: string;
+}
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  filterStatus,
+}) => {
+  const filteredTodos = todos.filter(todo => {
+    switch (filterStatus) {
+      case Status.Active:
+        return !todo.completed;
+      case Status.Completed:
+        return todo.completed;
+      case Status.All:
+      default:
+        return todo;
+    }
+  });
```

Comment:
It's better to filter in the parent component, so your `TodoList` doesn't have any logic, just render todos